Sources/SwiftSyntaxBuilder/Tokens.swift.gyb

```diff
@@ -22,27 +22,24 @@ import SwiftSyntax
 
 /// Namespace for commonly used tokens with default trivia.
 enum Tokens {
-  // MARK: Keywords
-
 % for token in SYNTAX_TOKENS:
 %   if token.is_keyword:
-  /// The `${token.text}` keyword
-  static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Keyword().withTrailingTrivia(.spaces(1))
+  /// The `${token.text.encode('utf-8').decode('unicode_escape')}` keyword
+  static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Keyword()
+%   elif token.text:
+  /// The `${token.text.encode('utf-8').decode('unicode_escape')}` token
+  static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Token()
+%   else:
+  static func `${lowercase_first_word(token.name)}`(_ text: String) -> TokenSyntax {
+    SyntaxFactory.make${token.name}(text)
+  }
+%   end
+%   if token.requires_leading_space:
+    .withLeadingTrivia(.spaces(1))
+%   end
+%   if token.requires_trailing_space:
+    .withTrailingTrivia(.spaces(1))
```

Comment:
This will break once we have a token with text that requires a leading or trailing space, because then we’d get

```swift
static func `${lowercase_first_word(token.name)}`(_ text: String) -> TokenSyntax {
  SyntaxFactory.make${token.name}(text)
}
.withLeadingTrivia(.spaces(1))
```

which doesn’t compile.

Reply:
I'm not sure that I understand?

So we have to move

```
%   if token.requires_leading_space:
     .withLeadingTrivia(.spaces(1))
 %   end
```
and
```
%   if token.requires_leading_space:
     .withLeadingTrivia(.spaces(1))
 %   end
```

inside so it look like

```
  /// The `${token.text.encode('utf-8').decode('unicode_escape')}` keyword
   static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Keyword()
%   if token.requires_leading_space:
     .withLeadingTrivia(.spaces(1))
 %   end
 %   elif token.text:
   /// The `${token.text.encode('utf-8').decode('unicode_escape')}` token
   static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Token()
 %   if token.requires_leading_space:
     .withLeadingTrivia(.spaces(1))
 %   end
 %   else:
   static func `${lowercase_first_word(token.name)}`(_ text: String) -> TokenSyntax {
     SyntaxFactory.make${token.name}(text)
%   if token.requires_leading_space:
     .withLeadingTrivia(.spaces(1))
 %   end
   }
```

Reply:
Yes, exactly.

Reply:
Got it. Should be pushed. 
I've run the `gyb` generation, to ensure nothing have changed.